fast_agave/tasks/sqs_tasks.py

```diff
@@ -43,15 +47,15 @@ async def start_task(*args, **kwargs) -> None:
             session = aiobotocore.session.get_session()
             async with session.create_client('sqs', region_name) as sqs:
                 for _ in count():
-                    response = await sqs.receive_message(
-                        QueueUrl=queue_url,
-                        WaitTimeSeconds=wait_time_seconds,
-                        VisibilityTimeout=visibility_timeout,
-                        AttributeNames=['ApproximateReceiveCount'],
-                    )
                     try:
+                        response = await sqs.receive_message(
+                            QueueUrl=queue_url,
+                            WaitTimeSeconds=wait_time_seconds,
+                            VisibilityTimeout=visibility_timeout,
+                            AttributeNames=['ApproximateReceiveCount'],
+                        )
                         messages = response['Messages']
-                    except KeyError:
+                    except Exception:
```

Comment:
esto podría funcionar, mi sugerencia es que se controle el error que representa la desconexión como en https://sentry.io/organizations/cuenca-mx/issues/3411902026/?project=6523122&query=is%3Aunresolved&statsPeriod=90d